controller/volume_controller_test.go

```diff
@@ -102,7 +103,6 @@ func (s *TestSuite) TestVolumeLifeCycle(c *C) {
 	// default replica and engine objects will be copied by copyCurrentToExpect
 	tc.copyCurrentToExpect()
 	tc.expectVolume.Status.State = longhorn.VolumeStateCreating
-	tc.expectVolume.Status.Robustness = longhorn.VolumeRobustnessUnknown
```

Comment:
Why removing this?

Reply:
Behavior of volume_controller changed, now when the volume is just created we create replicas and engine, set `vol.Status.State = longhorn.VolumeStateCreating` and returns: [link](https://github.com/longhorn/longhorn-manager/blob/a5b67e5ec57ec38be0d50cdbe77428d924f1b2fb/controller/volume_controller.go#L1381C2-L1381C2). We set volume reobusnees until the next sync.

Reply:
I see. When the volume is updated from `creating` to `detaching` (in the next test case), this volume robustness field is set to `longhorn.VolumeRobustnessUnknown`